cmd/kubeless/function/function.go

```diff
@@ -33,7 +33,7 @@ import (
 	kubelessApi "github.com/kubeless/kubeless/pkg/apis/kubeless/v1beta1"
 	"github.com/kubeless/kubeless/pkg/client/clientset/versioned"
 	"github.com/spf13/cobra"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
the change in this line is not necessary (you are still using `v1`)

Reply:
Great! Thank you :)

Reply:
It was autoimported this way. But I've removed it